tests/test_importing_geometries.py

```diff
@@ -0,0 +1,141 @@
+import os
+
+import pytest
+
+from ansys.mapdl.core.errors import MapdlInvalidRoutineError, MapdlRuntimeError
+from conftest import ON_CI
+
+PATH = os.path.dirname(os.path.abspath(__file__))
+
+# Files with CADs
+CADs_path = os.path.join(PATH, "test_files", "CAD_formats")
+
+
+####################################################################
+# Reading in all the supported formats
+
+
+def geometry_test_is_correct(mapdl, geometry=None):
+    if geometry == "iges":
+        entities = [0, 8, 36, 44]
+    else:
+        entities = [1, 8, 18, 12]
+
+    assert (
+        mapdl.geometry.n_keypoint == entities[3]
+    ), f"The number of keypoints ({mapdl.geometry.n_keypoint}) is not correct."
+    assert (
+        mapdl.geometry.n_line == entities[2]
+    ), f"The number of lines ({mapdl.geometry.n_line}) is not correct."
+    assert (
+        mapdl.geometry.n_area == entities[1]
+    ), f"The number of areas ({mapdl.geometry.n_area}) is not correct."
+    assert (
+        mapdl.geometry.n_volu == entities[0]
+    ), f"The number of volumes ({mapdl.geometry.n_volu}) is not correct."
+    return True
+
+
+def clear_wkdir_from_cads(mapdl):
+    """Cleaning files because we don't want to reuse the ANF files"""
```

Comment:
@germa89 perhaps you could use a pytest plugin to change the mapdl working directory, and clear/remove the old one. so that this can be done automatically for any sort of cleanup

Reply:
True. I will do!